### PR TITLE
Pd-453 update alerts list

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 1]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
 78|PoolExpandInProgress|NOTIFY|POOL|Triggered when a pool expand operation starts.|
 79|PoolExpandSuccessful|NOTIFY|POOL|Triggered when a pool expand operation succeeds.|
 80|PoolExpandFailed|ALARM|POOL|Triggered when a pool expand operation fails.|
+86|StoragelessToStorageNodeTransitionFailure|ALARM|NODE|Node failed to transition from storageless to storage.|
+87|StoragelessToStorageNodeTransitionSuccess|NOTIFY|NODE|Node transition from storageless to storage successful.|
+89|ClusterLicenseUpdated
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]: @adityadani I didn't see these in the list

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 2]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+105| KvdbConnectionFailed|ALARM|NODE|Triggered if Portworx fails to connect to the KVDB.|
+106| InternalKvdbSetupFailed|ALARM|NODE|Triggered if Portworx fails to set the Internal KVDB during installation.|
+107| OciMonImagePullFailed|ALARM|NODE|Triggered if Portworx fails to pull the OCI Monitor image during installation.|
+108| OciMonPrePostExecutionFailed|ALARM|NODE|Triggered if Portworx fails to execute pre or post installation tasks.|
+109| OciMonMountValidationFailed|ALARM|NODE|Triggered if Portworx fails to verify that the OCI Monitor was sucessfully mounted during instllation. |
+110| OciMonSchedulerInitializationFailed|ALARM|NODE|Triggered if Portworx fails to initialize the OCI Monitor scheduler during installation. |
+111| OciMonServiceControlsInitializationFailed|ALARM|NODE|Triggered if Portworx fails to initialize the OCI Monitor service controls during installation.|
+112| OciMonSchedulerInstallFailed|ALARM|NODE|Triggered if Portworx fails to install the OCI Monitor scheduler during Portworx installation.|
+113| MissingInputArgument|ALARM|NODE|Triggered if there's a missing input argument.|
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]: @adityadani What's the context for this? Is Portworx missing an input argument as part of installation?

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 3]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+86|StoragelessToStorageNodeTransitionFailure|ALARM|NODE|Triggered when a node fails to transition from a storageless type to a storage type.|
+87|StoragelessToStorageNodeTransitionSuccess|NOTIFY|NODE|Triggered when a node transitions from a storageless type to a storage type successfully.|
+89|ClusterLicenseUpdated
+90|LicenseExpired
+91|LicenseLeaseExpiring
+92|LicenseLeaseExpired
+93|LicenseServerDown
+94|FloatingLicenseSetupError
+95|NFSServerUnhealthy|WARNING|NODE|Triggered when the NFS server on this node is unhealthy.|
+96|FileSystemDependency|ALARM|NODE|Triggered during Portworx installation if there's a filesystem dependency failure. |
```
Comment (nathan-px) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]: @adityadani I assumed most of these are related to installation. That's the case, right?

Comment (adityadani) [reply content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]: Yes the FileSystemDependency and RebootRequired alerts are raised during Portworx installation or upgrade

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 4]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+97|RebootRequired|ALARM|NODE| Triggered when a node requires a reboot.|
+98|TempFileSystemInitialization|ALARM|NODE|Triggered during Portworx installation if a node fails to initialize a temporary filesystem.|
+99|UnsupportedKernel|ALARM|NODE|Triggered during a Portworx installation if the node contains a kernel that is not supported by Portworx.|
+100| InvalidDevice|ALARM|NODE|Invalid device|
```
Comment (adityadani) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]:
```suggestion
100| InvalidDevice|ALARM|NODE|Triggered during Portworx installation if an invalid device is provided to Portworx as a storage device.|
```

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 5]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+101| NfsDependencyInstallFailure|ALARM|NODE|Triggered during Portworx installation if Portworx cannot install the NFS service.|
+102| NfsDependencyNotEnabled|ALARM|NODE|Triggered during Portworx installation if Portworx cannot enable the NFS service.|
+103| LicenseCheckFailed|ALARM|NODE|Triggered if a node fails a license check.|
+104| PortworxStoppedOnNode|WARNING|NODE|Triggered if Portworx is stopped on this node.|
```
Comment (adityadani) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]:
```suggestion
104| PortworxStoppedOnNode|WARNING|NODE|Triggered if Portworx is stopped on a node.|
```

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 6]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+105| KvdbConnectionFailed|ALARM|NODE|Triggered if Portworx fails to connect to the KVDB.|
+106| InternalKvdbSetupFailed|ALARM|NODE|Triggered if Portworx fails to set the Internal KVDB during installation.|
```
Comment (adityadani) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]: No need of installation here.
```suggestion
106| InternalKvdbSetupFailed|ALARM|NODE|Triggered if Portworx fails to setup Internal KVDB on a node.|
```

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 7]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+107| OciMonImagePullFailed|ALARM|NODE|Triggered if Portworx fails to pull the OCI Monitor image during installation.|
```
Comment (adityadani) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]: We renamed all the OciMon alerts yesterday
```suggestion
107| PortworxMonitorImagePullFailed|ALARM|NODE|Triggered if Portworx fails to pull Portworx images during installation.|
```

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 8]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+108| OciMonPrePostExecutionFailed|ALARM|NODE|Triggered if Portworx fails to execute pre or post installation tasks.|
```
Comment (adityadani) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]:
```suggestion
108| PortworxMonitorPrePostExecutionFailed|ALARM|NODE|Triggered if Portworx fails to execute pre or post installation tasks.|
```

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 9]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+109| OciMonMountValidationFailed|ALARM|NODE|Triggered if Portworx fails to verify that the OCI Monitor was sucessfully mounted during instllation. |
```
Comment (adityadani) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]:
```suggestion
109| PortworxMonitorMountValidationFailed|ALARM|NODE|Triggered if Portworx fails to validate mounts provided to Portworx container during installation. |
```

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 10]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+110| OciMonSchedulerInitializationFailed|ALARM|NODE|Triggered if Portworx fails to initialize the OCI Monitor scheduler during installation. |
```
Comment (adityadani) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]:
```suggestion
110| PortworxMonitorSchedulerInitializationFailed|ALARM|NODE|Triggered if Portworx fails to initialize connection with scheduler during installation. |
```

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 11]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+111| OciMonServiceControlsInitializationFailed|ALARM|NODE|Triggered if Portworx fails to initialize the OCI Monitor service controls during installation.|
```
Comment (adityadani) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]:
```suggestion
111| PortworxMonitorServiceControlsInitializationFailed|ALARM|NODE|Triggered if Portworx fails to initialize the service controls during installation.|
```

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md [hunk 12]
```diff
@@ -103,3 +103,30 @@ Each alert has a severity from one of the following levels:
...
+112| OciMonSchedulerInstallFailed|ALARM|NODE|Triggered if Portworx fails to install the OCI Monitor scheduler during Portworx installation.|
```
Comment (adityadani) [inline content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]: The name of this alert has changed.
```suggestion
112| PortworxMonitorInstallFailed|ALARM|NODE|Triggered if Portworx installation fails.|
```

File: content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md (hunk 2 above)
Comment (adityadani) [reply content/install-with-other/operate-and-maintain/monitoring/portworx-alerts.md]: Yes this is triggered as a part of installation. Also we added another alert here:
```suggestion
113| MissingInputArgument|ALARM|NODE|Triggered if there's a missing input install argument.|
114|PortworxMonitorImagePullInProgress|NOTIFY|NODE|Triggered when Portworx is pulling and extracting images during installation or upgrade.|
```

**Pull request merged**